pom.xml

```diff
@@ -102,18 +102,6 @@
 
   <build>
     <plugins>
-      <plugin>
-        <groupId>org.eclipse.tycho</groupId>
-        <artifactId>tycho-source-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>feature-source</id>
-            <goals>
-              <goal>feature-source</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
       <plugin>
```

Comment:
attach-p2-metadata is there only for the source feature. Please remove it.

Reply:
Thanks!